lib/galaxy/config/template.py

```diff
@@ -1,18 +1,17 @@
 """Render customizable templates e.g. emails with Jinja.
```

Comment:
Nitpicking, but maybe this filename should be `templates.py` instead of `template.py` ?